README.md

```diff
@@ -269,9 +269,9 @@ kubectl delete pod vcp-manager --namespace vmware
 
 kubectl delete daemonset vcp-daementset --namespace vmware
 
-kubectl delete ThirdPartyResources vcp-status.vmware.com
+kubectl delete customresourcedefinitions vcp-status.vmware.com
 
-kubectl delete ThirdPartyResources vcp-summary.vmware.com
+kubectl delete customresourcedefinitions vcp-summary.vmware.com
```

Comment:
We should put note in  the  `README.md`  for 1.7 release to use `ThirdPartyResources` instead of `customresourcedefinitions` for cleanup commands.